docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  prisma:
+    image: prismagraphql/prisma:1.15
+    restart: always
+    ports:
+      - "4466:4466"
+    environment:
+      PRISMA_CONFIG: |
+        port: 4466
+        # managementApiSecret: asdfasdjkl;asdfklj1234345asfsldjf12354
+        databases:
+          default:
+            connector: postgres
+            host: postgres
+            port: 5432
+            user: prisma
+            password: prisma
+            migrations: true
+  postgres:
+    image: postgres
```

Comment:
Any special reason you're going for postgres? Better for graphql/prisma somehow?